web/static/js/components/idea_edit_form.jsx

```diff
@@ -4,16 +4,23 @@ import * as AppPropTypes from "../prop_types"
 class IdeaEditForm extends Component {
   constructor(props) {
     super(props)
-    this.state = { ideaBody: props.idea.body }
+    this.state = {
+      ideaBody: props.idea.body,
+      ideaCategory: props.category,
```

Comment:
wait, shouldn't the `category` be on the `idea` that gets passed as a prop?

Reply:
that's how we're getting the `body`

Reply:
And in fact, I think you safely set the `idea`  on state

Reply:
yep - good point!